pkg/agent/controller/networkpolicy/allocator_test.go

```diff
@@ -222,7 +223,8 @@ func TestWorker(t *testing.T) {
 				}
 				return false, nil
 			}
-			if err := wait.PollImmediate(time.Millisecond, time.Millisecond*10, conditionFunc); err != nil {
+
+			if err := wait.PollImmediate(time.Millisecond, time.Second, conditionFunc); err != nil {
```

Comment:
does it make sense to poll every millisecond?

Also shouldn't we check that the rule is not deleted *immediately* when we call `forgetRule` (which would probably require increasing the delete interval a bit so that the test is reliable)?

Reply:
If we poll every millisecond, we could check whether the rule is immediately deleted or it stayed in the async cache for expected amount of time. 
I was checking for elapsed time (L235) relative to the time just before `forgetRule` is being called to see if the rule is in the cache for at least `allocator.deleteInterval`. Previously I used hardcoded  value of 5 ms. Now I changed to `a.deleteInterval'

Reply:
I still think we should have some longer time intervals to reduce the probability of flakes in CI, e..g in the 100s of ms

Reply:
done